src/test/groovy/com/vanniktech/android/junit/jacoco/GenerationTest.groovy

```diff
@@ -269,15 +271,22 @@ class GenerationTest {
             }
 
             assert reports.xml.enabled
-            assert reports.xml.destination.toString() == project.buildDir.absolutePath + "/reports/jacoco/${flavor}${buildType.capitalize()}/jacoco.xml"
+            assert reports.xml.destination.toPath() == Paths.get(project.buildDir.absolutePath, "/reports/jacoco/${flavor}${buildType.capitalize()}/jacoco.xml")
```

Comment:
Some tests were failing on Windows due to the different path separator. Made them system independent now.